src/Microsoft.DotNet.Build.Tasks/VersionTools/BaseDependenciesTask.cs

```diff
@@ -16,6 +16,16 @@ namespace Microsoft.DotNet.Build.Tasks.VersionTools
 {
     public abstract class BaseDependenciesTask : Task
     {
+        protected static string RawUrlMetadataName = "RawUrl";
```

Comment:
Yes I always find protected constants a little odd which is why I suggested internal. protected would have been OK as well if the users are limited to just derived types.